test/parallel/test-blob.js

```diff
@@ -228,8 +230,44 @@ assert.throws(() => new Blob({}), {
       code: 'ERR_INVALID_ARG_VALUE',
     });
   });
+
+  try {
+    const overSizedArray = [new Uint8Array(0xffffffff), [1]];
+    assert.throws(() => new Blob(overSizedArray), {
+      code: 'ERR_BUFFER_TOO_LARGE',
+    });
+  } catch (e) {
+    if (e.message !== 'Array buffer allocation failed') throw e;
+    common.skip(
+      'Insufficient memory on this platform for oversized buffer test.'
+    );
```

Comment:
Would this fix the crash?
```suggestion
    void new Blob([new Uint8Array(0xffffffff), [1]]);
  } catch (e) {
    if (e.message === 'Array buffer allocation failed') {
      common.skip(
        'Insufficient memory on this platform for oversized buffer test.'
      );
    } else {
      assert.strictEqual(e.code, 'ERR_BUFFER_TOO_LARGE');
    }
```

Reply:
Thank you for your suggestion.
It seems that CI error occured because mustCall function is skipped in same file.
I put the test that can be skipped in another file.